### ISSUE TITLE
Removals & renames don't trigger a recompile

### ISSUE THREAD

**Issue by kishorevaishnav** [livebud/bud#94]
Hi @matthewmueller, First of all, Thanks for building this bud !!

I thought of learning this by giving a try, so here is an understanding question:

So, I was able to install the bud, create the app, was able to see bud home page with the nice splash. Now I tried this command `bud new controller photos show index` and then was able to see the index page in the browser with the nice title. All good until here.

Now, I wanted to rename the `photos` to `photo`, so I thought of deleting the folders/files and retype `bud new...` and deleted the `photos` directory present in both the `controllers` & `views` directory & did `go mod tidy` and it ended up showing this error:

```
❯ go mod tidy
go: finding module for package github.com/kishorevaishnav/shinyroll/controller/photos
github.com/kishorevaishnav/shinyroll/bud imports
        github.com/kishorevaishnav/shinyroll/bud/.app/program imports
        github.com/kishorevaishnav/shinyroll/bud/.app/web imports
        github.com/kishorevaishnav/shinyroll/bud/.app/controller imports
        github.com/kishorevaishnav/shinyroll/controller/photos: no matching versions for query "latest"
```

I tried to find the word photos in the entire directory and couldn't find any word of it, rather than fixing this or reverting the code, would like to understand where is this cached? & what is the logic behind it. If you can give me some insight and direct me some files to understand will be of great help.

P.S. Apologize if this shouldn't be posted here - not sure where to ask these kind of questions.

**Comment by kishorevaishnav** [comment]
Never mind - I was able to find the bud directory. Ignore this issue.

**Comment by matthewmueller** [comment]
Hey thanks, for the issue, I'd like to re-open this because I also noticed this bug recently. The watcher should be able to notice file removals / renames and re-compile

The workaround right now is to remove the bud directory and restart bud.

**Comment by matthewmueller** [comment]
I'm hoping this is fixed by #123. I'll close and we can re-open if we encounter this again.
